pkgs/applications/science/biology/seqtk/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, zlib
+, libdeflate
+, isa-l
+}:
+
+stdenv.mkDerivation rec {
+  pname = "seqtk";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "lh3";
+    repo = "${pname}";
+    rev = "v${version}";
+    hash = "sha256-1Hw/lnoFQumuEJg1n2C6vnWkBa+VLiEiDrosghSm360=";
+  };
+
+  buildInputs = [ zlib libdeflate isa-l ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp $pname $out/bin
+  '';
+
+  meta = with lib; {
+    description = "Toolkit for processing sequences in FASTA/Q formats";
+    license = licenses.mit;
+    homepage = "https://github.com/lh3/seqtk";
+    platforms = platforms.x86_64;
```

Comment:
Are you sure that this won't run on ARM and will run on everything including Linuxes and BSDs?

Reply:
This should probably include at least linux or unix.